diff --git a/integration_tests/tests/jinja_helpers/test_slugify.sql b/integration_tests/tests/jinja_helpers/test_slugify.sql
index 7d07ec4..3dcc335 100644
--- a/integration_tests/tests/jinja_helpers/test_slugify.sql
+++ b/integration_tests/tests/jinja_helpers/test_slugify.sql
@@ -1,9 +1,11 @@
 with comparisons as (
   select '{{ dbt_utils.slugify("!Hell0 world-hi") }}' as output, 'hell0_world_hi' as expected
+  from SYS_CALENDAR.CALENDAR where day_of_calendar = 1
   union all
   select '{{ dbt_utils.slugify("0Hell0 world-hi") }}' as output, '_0hell0_world_hi' as expected
+  from SYS_CALENDAR.CALENDAR where day_of_calendar = 1
 )
 
 select * 
 from comparisons
-where output != expected
\ No newline at end of file
+where output <> expected
\ No newline at end of file
